### PR TITLE
tiledbsoma 1.4.0

### PR DESCRIPTION
**Pull request by johnkerl**
Following

* https://github.com/TileDB-Inc/tiledbsoma-feedstock/pull/36
* https://github.com/single-cell-data/TileDB-SOMA/wiki/Branches-and-releases

### CODE REVIEW

**Comment by johnkerl** [comment]
"Queued" for 3+ hours; not sure what's going on ...

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,20 +13,20 @@ package:
   name: {{ name }}
   version: {{ version }}
 
-source:
-  url: https://github.com/single-cell-data/TileDB-SOMA/archive/{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+#source:
```
Comment (Shelnutt2) [inline recipe/meta.yaml:16]: This section for source + url + sha256 needs to be uncommented.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {% set name = "tiledbsoma" %}
-{% set version = "1.3.0" %}
-{% set version_r = "0.99.1" %}
-{% set sha256 = "1fb5bf86440a47b724ffb083143bda8a26d79e036336bac57ac7c1feaa5d4b9d" %}
+{% set version = "1.4.0" %}
+{% set version_r = "1.4.0" %}
```
Comment (Shelnutt2) [inline recipe/meta.yaml:3]: Shall we replace the `version_r` and just use `version` now?

**Review by Shelnutt2 — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 2 above)
Comment (johnkerl) [reply recipe/meta.yaml]: yes; good call definitely

File: recipe/meta.yaml (hunk 1 above)
Comment (johnkerl) [reply recipe/meta.yaml]: that was an embarrassingly dumb mistake on my part 😐

**Comment by johnkerl** [comment]
@Shelnutt2 all checks are passing

**Review by Shelnutt2 — APPROVED** [review]

**Pull request merged**